dotnet-client-libraries/BenchmarkDotNetWithUnitTests/Tests/BenchmarkFixture.cs

```diff
@@ -0,0 +1,32 @@
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Order;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNetWithUnitTests;
+
+namespace Tests;
+
+public class BenchmarkFixture : IDisposable
+{
+    private Summary? _benchmarkSummary;
+
+    public BenchmarkFixture()
+    {
+        var config = new ManualConfig
+        {
+            SummaryStyle = SummaryStyle.Default.WithMaxParameterColumnWidth(100),
+            Orderer = new DefaultOrderer(SummaryOrderPolicy.FastestToSlowest),
+            Options = ConfigOptions.Default
+        };
+
+        _benchmarkSummary = BenchmarkRunner.Run(typeof(StringConcatBenchmarks), config);
```

Comment:
Here instead of using `typeof` we can just write `BenchmarkRunner.Run<StringConcatBenchmarks>(config)` as we do in the Program.cs. I know it is more of a style issue, but this keeps the code consistent between the unit tests and the console application.